### DISCUSSION TITLE
Imports errors

### DISCUSSION THREAD

**Discussion by panditdhamdhere** [Cyfrin/foundry-full-course-cu#2645] [General]
![image](https://github.com/user-attachments/assets/8d839803-0baa-469c-b8c4-6d07621c2e0a)
can anyone help why happening this inside my vs code even though installing everything correctly

**Comment by EngrPips** [comment]
Hello @panditdhamdhere, Have you set your `remappings` properly?

**Reply by Shiva-Sai-ssb** [reply]
@EngrPips, it's about importing from the `forge-std` library, right? 
Why are `remappings` needed, or am I missing something?

**Reply by EngrPips** [reply]
If you have imported the library and didn't set `remappings` sometimes you might need to use `lib/forge-std/src/Script.sol` to import the `Script.sol` file as `forge-std/Script.sol` might not work. But generally, `forge-std/Script.sol` always tends to work without remappings.

**Reply by Shiva-Sai-ssb** [reply]
Oh, okay.👍

**Comment by Shiva-Sai-ssb** [comment]
@panditdhamdhere, try reopening VS Code. 
Also, what do you get when running `forge build`? Does it run successfully or throw any errors?

**Reply by panditdhamdhere** [reply]
yes sir! actually it compile and deploys too.

**Comment by panditdhamdhere** [comment]
I am getting this another error!
I set private key correctly ran `source .env ` echo as well but on deployment time it is throwing error 
![image](https://github.com/user-attachments/assets/416f9dab-6cea-4473-9ece-3850e3a854f5)

**Reply by EngrPips** [reply]
Read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2299

**Reply by panditdhamdhere** [reply]
@EngrPips this is really helpful!

**Reply by EngrPips** [reply]
I am glad it helped.
